Stellar-contract-config-setting.x

```diff
@@ -158,26 +158,29 @@ enum ContractCostType {
 struct ContractCostParamEntry {
     // use `ext` to add more terms (e.g. higher order polynomials) in the future
     ExtensionPoint ext;
-    
+
     int64 constTerm;
     int64 linearTerm;
 };
 
 struct StateExpirationSettings {
     uint32 maxEntryExpiration;
     uint32 minTempEntryExpiration;
-    uint32 minRestorableEntryExpiration;
+    uint32 minPersistentEntryExpiration;
     uint32 autoBumpLedgers;
 
     // rent_fee = wfee_rate_average / rent_rate_denominator_for_type
-    int64 restorableRentRateDenominator;
+    int64 persistentRentRateDenominator;
     int64 tempRentRateDenominator;
 
-    union switch (int v)
-    {
-    case 0:
-        void;
-    } ext;
+    // max number of entries that emit expiration meta in a single ledger
+    uint32 maxEntriesToExpire;
```

Comment:
What happens if number of expiring entries exceeds this? They still all expire but some of them don't emit meta?